cmd/karavictl/cmd/role_create_test.go

```diff
@@ -58,7 +58,7 @@ func TestRoleCreateHandler(t *testing.T) {
 
 		cmd := NewRootCmd()
```

Comment:
Out of curiosity, why did we do a NewRootCmd() here instead of NewRoleCreateCmd()? Then we don't need to add the role, create, into the Args?